docs/content/usage/packages/pypi.en-us.md

```diff
@@ -35,11 +34,11 @@ username = {username}
 password = {password}
 ```
 
-| Placeholder  | Description |
-| ------------ | ----------- |
-| `owner`      | The owner of the package. |
-| `username`   | Your Gitea username. |
-| `password`   | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
+| Placeholder  | Description                                                                                                                                    |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| `owner`    | The owner of the package.                                                                                                                      |
+| `username` | Your Gitea username.                                                                                                                           |
+| `password` | Your Gitea password. If you are using 2FA or OAuth use a[personal access token](development/api-usage.md#authentication) instead of the password. |
```

Comment:
```suggestion
| `password` | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
```